<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-V6C11AH6" failures="1" tests="1" name="Default test(failed)" time="13.402" errors="0" timestamp="2022-03-13T09:14:02 IST">
  <testcase classname="testcases.LoginTest" name="extData" time="6.391">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String, String)&amp;quot; because &amp;quot;this.test&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String, String)" because "this.test" is null
at Pages.LoginPage.LoginFunction(LoginPage.java:37)
at testcases.LoginTest.extData(LoginTest.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- extData -->
  <testcase classname="testcases.LoginTest" name="PassedLogin" time="0.0">
    <ignored/>
  </testcase> <!-- PassedLogin -->
  <testcase classname="testcases.LoginTest" name="FailedLogin" time="0.0">
    <ignored/>
  </testcase> <!-- FailedLogin -->
  <testcase classname="testcases.LoginTest" name="paramTest" time="0.0">
    <ignored/>
  </testcase> <!-- paramTest -->
</testsuite> <!-- Default test(failed) -->
